cmd/rm-main.go

```diff
@@ -121,17 +133,36 @@ func checkRmSyntax(ctx *cli.Context) {
 	isForce := ctx.Bool("force")
 	isRecursive := ctx.Bool("recursive")
 	isStdin := ctx.Bool("stdin")
-
+	isDangerous := ctx.Bool("dangerous")
+	isNamespaceRemoval := false
+	for _, url := range ctx.Args() {
+		// clean path for aliases like s3/
+		url = filepath.ToSlash(filepath.Clean(url))
```

Comment:
interestingly UNC path using  `/` is working in go1.9.2 by breaking the UNC path specification.

Reply:
It is not breaking @balamurugana they automatically convert it internally properly. Before any Win32 calls are made `os` package converts them to necessary values. 

FWIW Win32 calls support `/` path separators as well.

Reply:
Check on windows how it behaves UNC path with `/`

Reply:
```
C:\GO\src\github.com\minio\mc [(0f4e134...) +0 ~1 -0 !]> mc mb a/b/c/d/e
Bucket created successfully `a/b/c/d/e`.
C:\GO\src\github.com\minio\mc [(0f4e134...) +0 ~1 -0 !]> tree a
Folder PATH listing for volume Windows 10
Volume serial number is 00000200 4286:2744
C:\GO\SRC\GITHUB.COM\MINIO\MC\A
└───b
    └───c
        └───d
            └───e
C:\GO\src\github.com\minio\mc [(0f4e134...) +0 ~1 -0 !]> mc rm a/b/c/d/e
Removing `a/b/c/d/e`.
C:\GO\src\github.com\minio\mc [(0f4e134...) +0 ~1 -0 !]>
C:\GO\src\github.com\minio\mc [(0f4e134...) +0 ~1 -0 !]> mc rm //network1/blah/blah
mc.exe: <ERROR> Failed to remove `//network1/blah/blah`. Requested file `/blah/blah` not found
C:\GO\src\github.com\minio\mc [(0f4e134...) +0 ~1 -0 !]> mc rm --r --force //network1/blah/blah
mc.exe: <ERROR> Failed to remove `//network1/blah/blah` recursively. open /blah/blah: The system canno
h specified.
```

Reply:
No surprise go applications are working cleanly.  See my above comment.  Just add a comment what exactly i mentioned.

Reply:
done.